IfExcercises.java

```diff
@@ -0,0 +1,71 @@
+import java.util.Scanner;
+
+public class IfExcercises {
+	public static void main(String[] args) {
+		/*Napisz program, który sprawdza dla trzech boków trójkąta a ,
+		b i c wprowadzonych z klawiatury, czy tworzą one trójkąt pro-
+		stokątny (zakładamy, że a > 0 , b > 0 , c > 0 ).
+		*/
+
+		Scanner scanner = new Scanner(System.in);
+
+		int a, b, c;
```

Comment:
do not declare variables before like this, declare them when they are actually used